packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -506,7 +477,7 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
       return 'PrismaClient'
     }
 
-    private getEngine(): Engine {
+    getEngine(): Engine {
```

Comment:
I am curios, why `private` removal?

Reply:
That's for being able to export the types automatically via `infer`, typescript complained about it. I don't think it had much value anyways.